config/config-feature-flags.yaml

```diff
@@ -90,3 +90,7 @@ data:
   # in the TaskRun/PipelineRun such as the source from where a remote Task/Pipeline
   # definition was fetched.
   enable-provenance-in-status: "false"
+  # Setting this flag to "true" enables spire integration with pipeline.
+  # This is an experimental feature and thus should still be considered
+  # an alpha feature.
+  enable-spire: "false"
```

Comment:
NIT: it would be nice to use a name associated with the feature from a Tekton user point of view, e.g. `enable-taskrun-attestation` or so

Reply:
+1. Agree that this should be named from a Tekton user point of view. 
In the near future we might have SPIRE attesting the results generated by other objects like Pipelines.
enable-taskrun-attestation would be incorrect then.
How about enable-non-falsifiable-provenance ?